src/ansys/fluent/core/quantity/dimensions.py

```diff
@@ -120,3 +125,22 @@ def units(self):
     def dimensions(self):
         """Dimensions list."""
         return self._dimensions
+
+
+class DimensionsError(ValueError):
+    """Custom dimensions errors."""
+
+    def __init__(self, err):
+        super().__init__(err)
+
+    @classmethod
+    def EXCESSIVE_PARAMETERS(cls):
+        return cls(
+            "Dimensions only accepts 1 of the following: (units) or (dimensions)."
+        )
+
+    @classmethod
+    def EXCESSIVE_DIMENSIONS(cls, len):
+        return cls(
+            f"`dimensions` must contain 9 values or less, currently there are {len}."
```

Comment:
Shall we write **D**imensions in both cases

Reply:
Added more clarification to error messages, Dimensions was referring to the class while `dimensions` is the parameter that accepts a list of dimensions.

https://github.com/ansys/pyfluent/blob/16bfd1f51b57625263ffefce7cf960c6691ef305/src/ansys/fluent/core/quantity/dimensions.py#L144-L154

Reply:
thanks for clarifying